noir-projects/noir-protocol-circuits/crates/types/src/constants.nr

```diff
@@ -182,7 +182,7 @@ global DEPLOYER_CONTRACT_INSTANCE_DEPLOYED_MAGIC_VALUE =
 
 // GAS DEFAULTS
 global DEFAULT_GAS_LIMIT: u32 = 1_000_000_000;
-global DEFAULT_TEARDOWN_GAS_LIMIT: u32 = 100_000_000;
+global DEFAULT_TEARDOWN_GAS_LIMIT: u32 = 12_000_000;
```

Comment:
There is no reason the teardown limit should be higher than the enqueued call limit

Reply:
It maybe doesn't make sense to change this here, but I did so while debugging a test failure and thought... why change it back?

Reply:
then maybe you can move the line past MAX_L2_GAS_PER_ENQUEUED_CALL and do

```
global DEFAULT_TEARDOWN_GAS_LIMIT  = MAX_L2_GAS_PER_ENQUEUED_CALL;
```
?